src/kcas.mli

```diff
@@ -427,13 +433,13 @@ module Xt : sig
 
   val update : xt:'x t -> 'a Loc.t -> ('a -> 'a) -> 'a
   (** [update ~xt r f] is equivalent to [let x = get ~xt r in set ~xt r (f x); x]
-      with the limitation that [f] must not and is not allowed to record
-      accesses to the transaction log. *)
+      with the limitation that [f] must not and is not allowed to access the
+      transaction log. *)
```

Comment:
Reworded the sentence here to avoid impression that `is_in_log` would be safe to use.